changelogs/fragments/951-ipa_user-add-userauthtype-param.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - "ipa_user - add ``userauthtype`` param (https://github.com/ansible-collections/community.general/pull/951)."
```

```suggestion
  - "ipa_user - add ``userauthtype`` option (https://github.com/ansible-collections/community.general/pull/951)."
```
according to the doc `option` is more preferable than `parameter` or `argument`